docs/software/requesting-tokens.md

```diff
@@ -64,17 +62,44 @@ and refresh tokens from OpenID Connect token providers.
     1. Enter a password to encrypt your local client profile.
        You'll need to remember this if you want to re-use this profile in subsequent sessions.
 
-1. Request a token using the client name that you used above with `oidc-gen`:
+### Requesting access tokens
 
+!!! note
+    You must first [register a new profile](#registering-an-oidc-profile).
+
+1. Request a token using the client profile that you used with `oidc-gen`:
+	
         :::console
-        docker exec -it my-agent oidc-token --aud="<SERVER AUDIENCE>" <CLIENT NAME>
+        docker exec -it my-agent oidc-token --aud="<SERVER AUDIENCE>" <CLIENT PROFILE>
+
 
-   For tokens used against an HTCondor-CE, set `<SERVER AUDIENCE>` to `<CE FQDN>:<CE PORT>`.
+    For tokens used against an HTCondor-CE, set `<SERVER AUDIENCE>` to  
+    `<CE FQDN>:<CE PORT>`.
 
-1. Copy the output of `oidc-token` into a file on the host where you need SciToken authentication, e.g. an HTCondor or
+  1. Copy the output of `oidc-token` into a file on the host where you need SciToken authentication, e.g. an HTCondor or
    XRootD client.
 
-### Using an RPM
+### Reloading an OIDC profile
+
+!!! note
+    Required after restarting the running container. You must have an existing [registered profile](#registering-an-oidc-profile).
+
+1. Reload profile:
+
+            :::console
+            docker exec -it my-agent oidc-add <CLIENT PROFILE>
+
+    1. If your existing container is not already running, start it:
+
+                :::console
+                docker start my-agent
```

Comment:
The ordering of the steps is broken here. The first command will fail if your container isn't running!

Also, the indentation will cause the `docker start my-agent` step to be a sub-bullet of the first and we don't want that